### PR TITLE
[23_28] Change Cursor Style when Rotate or Zoom

### PR DESCRIPTION
**Pull request by Oyyko**
## Why you open this Pull Request?

In the past PR #815 , we add a feature that the cursor style will change in MOVE MODE. Now, this feature is usable in ROTATE and ZOOM MODE too.

## What work have you done in the current Pull Request?

- [x] add cursor style changing function in zoom and rotate mode.

### CODE REVIEW

File: TeXmacs/progs/graphics/graphics-env.scm [hunk 1]
```diff
@@ -485,17 +485,19 @@
   (set-cursor-style-origin style)
   (set! the-current-cursor-style style))
 
+;; modes in which the cursor should become hand-style, see graphics-main.scm too.
+(define hand-modes '((tuple "group-edit" "move") (tuple "group-edit" "rotate") (tuple "group-edit" "zoom")))
```
Comment (darcy-shen) [inline TeXmacs/progs/graphics/graphics-env.scm:489]: code format

File: TeXmacs/progs/graphics/graphics-main.scm [hunk 2]
```diff
@@ -854,13 +854,16 @@
       (set! mode (cadr mode)))
   (== mode (graphics-mode)))
 
+;; modes in which the cursor should become hand-style, see graphics-env.scm too.
+(define hand-modes '((group-edit move) (group-edit zoom) (group-edit rotate)))
```
Comment (darcy-shen) [inline TeXmacs/progs/graphics/graphics-main.scm:858]: code format

File: devel/23.tm [hunk 3]
```diff
@@ -380,6 +380,19 @@
       \;
     </cell>|<\cell>
       <value|dis>
+    </cell>>|<row|<\cell>
```
Comment (darcy-shen) [inline devel/23.tm:383]: project management should be in a separate pull request

File: TeXmacs/progs/graphics/graphics-env.scm (hunk 1 above)
Comment (Oyyko) [reply TeXmacs/progs/graphics/graphics-env.scm]: OK!

File: TeXmacs/progs/graphics/graphics-main.scm (hunk 2 above)
Comment (Oyyko) [reply TeXmacs/progs/graphics/graphics-main.scm]: OK!

File: devel/23.tm (hunk 3 above)
Comment (Oyyko) [reply devel/23.tm]: OK!

**Comment by darcy-shen** [comment]
It seems there are no suitable cursor styles for rotate or zoom?

**Comment by darcy-shen** [comment]
It is acceptable to use the openhand and closehand.

**Review by darcy-shen — APPROVED** [review]
Tested on Windows
